composer.json

```diff
@@ -10,9 +10,12 @@
     "license": "MIT",
     "authors": [
         {
-            "name": "Will Pillar",
-            "email": "will.pillar@graze.com"
+            "name": "Graze Developers",
+            "email": "developers@graze.com"
         }
     ],
-    "require": {}
+    "require": {},
+    "require-dev": {
+        "squizlabs/php_codesniffer": "^3.0"
```

Comment:
i think i ran composer locally so i could check in phpstorm.

Thats probably a good point, maybe i'll setup some installation and linting